### ISSUE TITLE
[Bug] Error loading history file

### ISSUE THREAD

**Issue by lorensr** [temporalio/vscode-debugger-extension#43]
<img width="516" alt="image" src="https://github.com/temporalio/vscode-debugger-extension/assets/251288/d6698454-cf28-45bc-bdb5-5c4fddb81f92">

<img width="780" alt="image" src="https://github.com/temporalio/vscode-debugger-extension/assets/251288/b5caa1f5-713e-4587-8f20-1fbc53cb6a36">

File downloaded from UI: https://www.dropbox.com/s/3j30ypo8w7ajwcd/e79b3473-5234-4046-be33-91c84a0af32a_events.json?dl=0

<img width="518" alt="image" src="https://github.com/temporalio/vscode-debugger-extension/assets/251288/63737df8-8942-445a-91d4-700450317900">

workflow code:

```ts
import { condition, defineSignal, proxyActivities, setHandler, sleep } from '@temporalio/workflow';
import type * as activities from './activities';

export const verifySignal = defineSignal<[]>('verify');

const { notifyHumanForVerification, collectFeedback } = proxyActivities<typeof activities>({
  startToCloseTimeout: '1 minute',
});

export async function humanVerificationWorkflow(task: string) {
  let verified = false;

  await notifyHumanForVerification(task);
  setHandler(verifySignal, () => {
    verified = true;
  });

  await condition(() => verified);
  await sleep('1 day');
  await collectFeedback();
}
```

**Comment by lorensr** [comment]
<img width="1577" alt="image" src="https://github.com/temporalio/vscode-debugger-extension/assets/251288/936bdeb0-8d8e-4a7f-8b02-d8746715ae65">

**Comment by bergundy** [comment]
Seems like an SDK update should solve this, I'll verify and update if needed.

**Comment by lorensr** [comment]
Fixed by #44

**Comment by abhishek-parative** [comment]
Hi I'm experiencing the same exact error running on version 1.0.2 of the extension
